src/modules/commander/commander.cpp

```diff
@@ -449,7 +449,25 @@ int commander_main(int argc, char *argv[])
 			} else if (!strcmp(argv[2], "accel")) {
 				calib_ret = do_accel_calibration(&mavlink_log_pub);
 			} else if (!strcmp(argv[2], "gyro")) {
-				calib_ret = do_gyro_calibration(&mavlink_log_pub);
+				/* send vehicle_command to trigger gyro calibration */
+				struct vehicle_command_s cmd = {
```

Comment:
This is good for moving them incrementally, but once commander has all calibration code removed I think we can get rid it.